spec/models/vote_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe Vote, type: :model do
+  it 'increments score' do
+    vote = FactoryGirl.create(:vote)
+    vote.increment
+    expect(vote.score).to eq 1
+  end
+  it 'decrements score' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.